tesseract_motion_planners/core/include/tesseract_motion_planners/simple/profile/simple_planner_fixed_size_plan_profile.h

```diff
@@ -53,12 +53,12 @@ class SimplePlannerFixedSizePlanProfile : public SimplePlannerPlanProfile
    */
   SimplePlannerFixedSizePlanProfile(int freespace_steps = 10, int linear_steps = 10);
 
-  CompositeInstruction generate(const PlanInstruction& prev_instruction,
-                                const MoveInstruction& prev_seed,
-                                const PlanInstruction& base_instruction,
-                                const Instruction& next_instruction,
-                                const PlannerRequest& request,
-                                const ManipulatorInfo& global_manip_info) const override;
+  virtual CompositeInstruction generate(const PlanInstruction& prev_instruction,
```

Comment:
The virtual will have to be removed to make clang-tidy happy. Is this needed for some reason by SWIG?